src/baguetteBox.js

```diff
@@ -384,6 +385,9 @@
         // Fill options object
         for (var item in defaults) {
             options[item] = defaults[item];
+            if (!options[item]) {
+                options[item] = defaults[item];
+            }
```

Comment:
I know this is an old PR, but can you explain what it's fixing? Maybe I'm missing something, but I don't see how this code changes anything by assigning the same value twice